Library/Formula/gmtk.rb

```diff
@@ -0,0 +1,26 @@
+class Gmtk < Formula
+  desc "The Graphical Models Toolkit for prototyping dynamic graphical models"
+  homepage "http://melodi.ee.washington.edu/gmtk"
+  url "http://melodi.ee.washington.edu/downloads/gmtk/gmtk-1.4.4.tar.gz"
+  sha256 "c6243f1b5c68910a4a0ea60a1b2285ede5ba771cfec3fa5ffa3699ecef6e87e0"
+
+  depends_on "homebrew/science/hdf5" => :optional
+  depends_on "wxmac" => :recommended
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--without-x",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"smoke_test.str").write("GRAPHICAL_MODEL smoke_test frame: 0 { variable: X { type: discrete observed 0:0 cardinality 2; conditionalparents: nil using DenseCPT(\"X0\"); } } chunk 0:0")
```

Comment:
Can we shrink this line at all, or extend it into a block? It's currently making the formula quite lengthy. We try to keep things as close to the 80 character width as possible for readability.